hyp3_metadata/__main__.py

```diff
@@ -0,0 +1,207 @@
+import copy
+from datetime import datetime
+from pathlib import Path
+from typing import List, Tuple
+
+from jinja2 import Environment, PackageLoader, StrictUndefined, select_autoescape
+from osgeo import gdal, osr
+
+from hyp3_metadata import __version__
+
+
+def get_environment() -> Environment:
+    env = Environment(
+        loader=PackageLoader('hyp3_metadata', '.'),
```

Comment:
I'd consider moving our templates down to `hyp3_metadata/templates/` rather than having them in the module root